src/workerd/api/util-test.c++

```diff
@@ -53,5 +63,51 @@ KJ_TEST("redactUrl can detect base64 ids") {
   expectUnredacted("https://domain/IThinkIShallNeverSee0/x"_kj);
 }
 
+KJ_TEST("readContentTypeParameter can fetch boundary parameter") {
+
+  // normal
+  expectCTypeParameter(
+    "multipart/form-data; boundary=\"__boundary__\""_kj,
+    "boundary"_kj,
+    "__boundary__"_kj
+  );
+
+  // multiple params
+  expectCTypeParameter(
+    "multipart/form-data; charset=utf-8; boundary=\"__boundary__\""_kj,
```

Comment:
I'd recommend a test that puts these in a different order (e.g. `multipart/form-data; boundary="__boundary__"; charset=utf-8`) and one that includes another entirely arbitrary unknown parameter that has no meaning and is ignored. (e.g. `multipart/form-data; foo=123; boundary="__boundary__")

Reply:
Done.